Kickstarter-iOS/Views/FundingGraphView.swift

```diff
@@ -255,30 +255,6 @@ private func slope(ofLine line: Line) -> CGFloat {
 private func dateToDayNumber(launchDate: TimeInterval,
```

Comment:
it might be nice to rename this swift 3 style, something like `days(from launchDate: Date, to currentDate: Date)`, but doubt you want to do that now...